setup.cfg

```diff
@@ -1,3 +1,50 @@
+[metadata]
+name = httpx
+version = attr: src.get_version
+url = https://github.com/encode/httpx
+license = BSD
+description = The next generation HTTP client.
+long_description = file: README.md, file: CHANGELOG.md
+long_description_content_type = text/markdown
+author = Tom Christie
+author_email = tom@tomchristie.com
+classifiers =
+    Development Status :: 3 - Alpha
+    Environment :: Web Environment
+    Intended Audience :: Developers
+    License :: OSI Approved :: BSD License
+    Operating System :: OS Independent
+    Topic :: Internet :: WWW/HTTP
+    Framework :: AsyncIO
+    Framework :: Trio
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+
+[options]
+python_requires = >=3.6
+packages = find:
+include_package_data = True
+zip_safe = False
```

Comment:
pretty sure this package is zip_safe. (it's not hard to be, you just use importlib.resources for package data)

Reply:
Unfortunately we need this flag — required by PEP 561 to include the py.typed marker used by type checkers for detecting type annotations.

Reply:
https://github.com/ethanhs/pep-561/blob/master/setup.py this example doesn't define zip safe

Reply:
https://mypy.readthedocs.io/en/latest/installed_packages.html#making-pep-561-compatible-packages

> If you use setuptools, you must pass the option zip_safe=False to setup(), or mypy will not be able to find the installed package.